phpmd.xml

```diff
@@ -9,26 +9,17 @@
   </description>
 
   <!-- Import common rulesets -->
-  <rule ref="rulesets/codesize.xml"/>
-  <rule ref="rulesets/design.xml"/>
+  <rule ref="rulesets/design.xml">
+    <exclude name="CouplingBetweenObjects"/>
```

Comment:
I don't see this in DrupalQualityChecker.

Reply:
Yes It's not in the current release. I have raised a PR on quality checker for the same.